Milestone 2: Portfolio middle section

dewslyse opened the pull request:
### Portfolio: mobile version

- Added 'Works' section
- Added 'About myself' section
- Implemented button interactions

dewslyse committed:
Add via uplead

dewslyse committed:
Add 'works' section

dewslyse committed:
Add via upload

dewslyse committed:
Add about section

dewslyse committed:
Add button states

dewslyse committed:
Update portfolio screenshot

dewslyse committed:
Remove unused styles

keroloussamy commented on index.html:
```diff
@@ -57,6 +57,132 @@ <h1 class="intro">I’m Selase 👋 Glad to see you!</h1>
+                <p class="details">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <div class="tags">
+                    <p class="tag">html</p>
+                    <p class="tag">css</p>
+                    <p class="tag">javascript</p>
+                </div>
```
Kindly use `ul` tag instead of `div` tag, and `li` tag instead of `p` tag, and make the same for every project technologies in every project. To improve your structure and semantics.

keroloussamy commented on index.html:
```diff
@@ -57,6 +57,132 @@ <h1 class="intro">I’m Selase 👋 Glad to see you!</h1>
+                my work and experience! If you like what you see and have a project you need coded, don’t hestiate
+                to contact me.
+            </p>
+            <p class="social-media">Let's connect</p>
```
Kindly use `h1-6` tags instead of `p` tag, it's a kind of title so it's more semantic to use headline tags. this one `<p class="social-media">Let's connect</p>`

keroloussamy commented on index.html:
```diff
@@ -57,6 +57,132 @@ <h1 class="intro">I’m Selase 👋 Glad to see you!</h1>
+                </li>
+            </ul>
+            <button class="btn btn-resume">Get my resume</button>
+            <ul class="skill-list">
+                <li class="skill">Languages<span><img src="./images/chevron.png" alt=""></span></li>
```
Kindly add the technologies part to your design, you don't have to make any functionality, just add the HTML and CSS for it.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/153520326-ae2554c1-0857-4646-ae60-bccc5a9a67f9.png) | ![image](https://user-images.githubusercontent.com/35535130/153520395-d281c18d-0388-4da5-aba2-78163f6e7635.png) |

keroloussamy commented on index.html:
```diff
@@ -57,6 +57,132 @@ <h1 class="intro">I’m Selase 👋 Glad to see you!</h1>
+
+        <section class="works wrapper">
+            <h2 class="sr-only">Works Section</h2>
+            <article class="card">
+                <img src="./images/Tonic.png" alt="" aria-hidden="true">
+                <h3 class="title color-caption">Tonic</h3>
```
[OPTIONAL] Kindly add the `alt` attribute, it's very useful if the image didn't load. it's a good practice.

dewslyse replied:
@keroloussamy, I agree with you. However in this case, I believe the images are mainly decorative. That's why the `alt` is empty and `aria-hidden` set to `true`

keroloussamy reviewed (changes requested):
Hi @dewslyse, 

Good job so far, You did great work 💯 🎉!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._

![](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966)

## Optional suggestions
_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

dewslyse committed:
Add via upload

dewslyse committed:
Resolve issues from pull request

- Replace div and p tags with <ul> and <li> respectively for .details
- Replace <p> with <h2> tag on class .social-media
- Add technologies part to design

dewslyse committed:
Resolve Stylelint errors

dewslyse committed:
Resolve Stylelint errors

dewslyse committed:
Merge branch 'add-work-about-section' of github.com:dewslyse/Portfolio-mv into add-work-about-section

gemmen29 reviewed (approved):
Hi @dewslyse, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![](https://media.giphy.com/media/mCIjCgs3nWQWfJZvPA/giphy-downsized-large.gif)

## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
